jmxtrans.sh

```diff
@@ -17,7 +17,7 @@ fi
 
 JAVA_HOME=${JAVA_HOME:-"/usr"}
 LOG_DIR=${LOG_DIR:-"."}
-LOG_FILE=${LOG_FILE:-"/dev/null"}
+STDOUT_REDIRECT_FILE=${STDOUT_REDIRECT_FILE:-"/dev/null"}
```

Comment:
You don't really need a LOG_FILE variable as log file name was already
there in lo4j.xml

Using that environment variable was hack to make broken logging working.
On Feb 12, 2015 3:56 PM, "Jon Stevens" <notifications@github.com> wrote:

> In jmxtrans.sh
> <https://github.com/jmxtrans/jmxtrans/pull/223#discussion_r24619382>:
>
> > @@ -17,7 +17,7 @@ fi
> >
> >  JAVA_HOME=${JAVA_HOME:-"/usr"}
> >  LOG_DIR=${LOG_DIR:-"."}
> > -LOG_FILE=${LOG_FILE:-"/dev/null"}
> > +STDOUT_REDIRECT_FILE=${STDOUT_REDIRECT_FILE:-"/dev/null"}
>
> breaks backwards compat
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jmxtrans/jmxtrans/pull/223/files#r24619382>.
>